pages/[id]/collectibles.js

```diff
@@ -96,6 +98,12 @@ const Collection = ({ userProfile, accountId }) => {
 		}
 	}
 
+	const onClickDisplay = (typeDisplay) => {
+		setTokens([])
+		fetchOwnerTokens()
+		setDisplay(typeDisplay)
+	}
+
 	const headMeta = {
```

Comment:
since ini ngeset tokensnya jadi empty array lagi, berarti ngefetch dari 0. `idNext`, `priceNext`, sama `hasMore` nitip di update juga ya ke initial valuenya

Reply:
oke, fixing gan